release-schema.json

```diff
@@ -73,6 +78,38 @@
           "uniqueItems": true
         }
       }
+    },
+    "RelativeValue": {
+      "title": "Relative value",
+      "description": "The value of the participation fee, as a percentage of a monetary value.",
+      "type": "object",
+      "required": [
+        "percentage",
+        "monetaryValue"
+      ],
+      "properties": {
+        "percentage": {
+          "title": "Percentage",
+          "description": "The percentage of the monetary value.",
+          "type": [
+            "number"
+          ],
+          "maximum": 1,
+          "minimum": 0,
```

Comment:
Noting that this differs from the suggestion in https://github.com/open-contracting/ocds-extensions/issues/81#issuecomment-1830975752 (0 to 100) but is in keeping with the modelling of percentages in the subcontracting extension so I think it's all good.

Reply:
it's also how `bid.statistic.value` says to provide percentages ("Percentages should be provided as a proportion of 1 (e.g. 10% = 0.1)"), and `operatorRevenueShare` from Contract Terms and `shareholders.shareholding` from Shareholders is modelled.

Reply:
Hmm, we probably need to edit many of these to add ", expressed as a decimal proportion between 0 and 1."

Can you make that change here and open an issue to update contractTerms, shareholders, subcontracting, bids, and ppp?

bids can be updated to:

> A total count should be expressed as an integer. A proportion should be expressed as a decimal (for example, 0.1) and not as a percentage (for example, 10).

and ppp to:

> , expressed as a decimal (for example, 0.1) and not as a percentage (for example, 10).

Can also delete the "further details" sentence while we're editing ppp.